.jenkins

```diff
@@ -90,7 +90,6 @@ pipeline {
                                 -DKokkos_ENABLE_CUDA=ON \
                                 -DKokkos_ENABLE_CUDA_LAMBDA=ON \
                                 -DKokkos_ENABLE_OPENMP=ON \
-                                -DKokkos_ENABLE_IMPL_NVHPC_AS_DEVICE_COMPILER=ON \
```

Comment:
Update the build name `CUDA-12.2-NVHPC` -> `CUDA-12.2-NVHPC-AS-HOST_COMPILER` or such?

Reply:
Sure, why not.